example/instruction_tuning_pipeline/README.md

```diff
@@ -0,0 +1,151 @@
+# Deltatuner Fine-tuning Example
+
+This example demonstrates how to finetune the pretrained large language model (LLM) with the instruction-following dataset by using the Deltatuner and some foundamental models. Giving Fine-tuned model the textual instruction, it will respond with the textual response. This example have been validated on the 4th Gen Intel® Xeon® Processors, Sapphire Rapids.
+
+## Prerequisite​
+
+### 1. Environment​
+- build the docker image
+```shell
+docker build -f docker/Dockerfile -t chatbot_finetune .
+```
+- create docker container
+```shell
+# in cpu server
+docker run -it --name chatbot \
+    --privileged --network host --ipc=host \
+    --device=/dev/dri \
+    -v /dev/shm:/dev/shm \
+    -v ~:/home/vmagent/app \
+    -w /home/vmagent/app  \
+    chatbot_finetune:latest \
+    /bin/bash
+
+# in gpu server
+docker run -it --name chatbot \
+    --privileged --network host --ipc=host \
+    --device=/dev/dri \
+    --runtime=nvidia \
+    -v /dev/shm:/dev/shm \
+    -v /mnt/DP_disk1/dataset:/home/vmagent/app/data \
+    -v /mnt/DP_disk1/yu:/home/vmagent/app \
+    -w /home/vmagent/app/  \
+    chatbot_finetune:latest \
+    /bin/bash
+```
+- Or you can direct create from the bare mental env
+```shell
+pip install -r requirements.txt
+```
+
+- Install the `deltatuner` python package
+```shell
+pip install deltatuner
+```
+
+### 2. Prepare the Model
```

Comment:
Please refer to ITREX if below are from there.

Reply:
ok, added.

Reply:
What are the changes? Can you only show the changed part here?

Reply:
I have changed to "This example was based upon [Intel® Extension for Transformers](https://github.com/intel/intel-extension-for-transformers/tree/main/workflows/chatbot/fine_tuning)."